string item
---
pr2_pick_contest/ItemModel model
